fix-38380: New composer : insert youtube link .

IlhemEssaadi opened the pull request:
ISSUES :  When pasting a YouTube link, adding text under the embed element is not possible.
FIX : the problem is solved by adding a video plugin to the social module and eliminating the embedsemantic and embedbase plugins.

houssemexo26 committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

IlhemEssaadi committed:
TASK-38380 : When pasting a YouTube link, a white margin appears and the cross button delete the typed text. (#1276)

ISSUES : When adding a new post in space, resizing the text area and adding text under the embed link is not possible.
FIX : The first problem is that we cannot resize the textarea it fixed by adding the resize plugin of the CKEDITOR ,and the second one is that we can't add text under the embedded link and this has been solved by adding a function allowing focus after the embedded link.

IlhemEssaadi committed:
fix-38380: New composer : insert youtube link .

ISSUES :  When pasting a YouTube link, adding text under the embed element is not possible.
FIX : the problem is solved by adding a video plugin to the social module and eliminating the embedsemantic and embedbase plugins.

ghost commented:
### CodeSee Review Map:
[![Review these changes using an interactive CodeSee Map](https://s3.us-east-2.amazonaws.com/maps.codesee.io/images/github/Meeds-io/social/1313/35486973/2ab03a65b81e9340e83edbe3a18ee6f638af402e.svg)](https://app.codesee.io/r/reviews?pr=1313&src=https%3A%2F%2Fgithub.com%2FMeeds-io%2Fsocial)

Review [in an interactive map](https://app.codesee.io/r/reviews?pr=1313&src=https%3A%2F%2Fgithub.com%2FMeeds-io%2Fsocial)

View [more CodeSee Maps](https://app.codesee.io/r/maps?src=https%3A%2F%2Fgithub.com%2FMeeds-io%2Fsocial)

#### Legend
![CodeSee Map Legend](https://app.codesee.io/Legend_v2.png)

rdenarie reviewed (changes requested):
Please fix conlfict, and rebase, thanks

rdenarie committed:
Task-54770 : Add secure option on cookie last_login_username (#1336)

Before this fix, the cookie last_login_username have not the secure option
This fix add the option

rdenarie reviewed (changes requested):
Can you resolv the conflict and rebase the PR ?
Thanks

IlhemEssaadi committed:
fix SonarCloud Code Analysis problem

rdenarie reviewed (changes requested):
can you resolve the conflict in this pr ?

IlhemEssaadi committed:
Merge branch 'feature/maintenance' into fix/38380
